recipes/mpitrampoline/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "MPItrampoline" %}
+{% set version = "5.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/eschnett/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: bdfb145b754cf13d38485808f79a5638d954dbe181e7bb53a1950c71deed6989
+
+build:
+  # Fortran compiler is not found on Windows
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib('c') }}
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
```suggestion
```

This is not a python package. There should be no need for any of these dependencies since the file list does not contain any python modules.